src/main/java/ch/njol/skript/expressions/ExprIndices.java

```diff
@@ -34,25 +34,33 @@
 import ch.njol.skript.lang.UnparsedLiteral;
 import ch.njol.skript.lang.Variable;
 import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.registrations.Comparators;
 import ch.njol.util.Kleenean;
 
-@Name("Indexes of List")
-@Description("Returns all the indexes of a list variable")
+@Name("Indices of List")
+@Description(
+	"Returns all the indices of a list variable, optionally sorted by their values.\n" +
```

Comment:
Aren't new arguments are used in annotation arguments instead of \n generally? Like `@Description("Returns all the ... their values.", "line2", "line3")` because most documentation annotations accept String... values.

Nevermind they accept String[] (not var-args), so you must add {}, i.e `@Description({"line1", "line2"})` etc., still better than \n in my opinion though, to not have explicit escape sequences / new line characters everywhere.

Reply:
Done :)